src/core/manager.c

```diff
@@ -1823,7 +1823,36 @@ int manager_load_unit(
         manager_dispatch_load_queue(m);
 
         *_ret = unit_follow_merge(*_ret);
+        return 0;
+}
+
+int manager_load_live_unit_or_warn(
```

Comment:
hmm, I wonder if we can find a better name for this than "live"... I mean, we already have "dead" and one could assume that "live" and "dead" where opposite states, but they aren't they are orthogonal...

Maybe use "startable" instead?

Reply:
English has 200k words, and it's still to find one that fits ;) Updated.